common/src/test/java/tachyon/conf/TachyonConfTest.java

```diff
@@ -64,204 +64,204 @@ public void beforeTests() {
   @Test
   public void testCommonDefault() {
     String tachyonHome = sDefaultTachyonConf.get(Constants.TACHYON_HOME);
-    Assert.assertTrue(tachyonHome != null);
-    Assert.assertTrue("/mnt/tachyon_default_home".equals(tachyonHome));
+    Assert.assertNotEquals(tachyonHome, null);
```

Comment:
yes, I think they are equivalent semantically. however, `assert(Not)Null` is more specific for readability, though.